modules/002-deckhouse/webhooks/validating/d8-cluster-configuration

```diff
@@ -72,45 +59,58 @@ EOF
     return 0
   fi
 
-  # defaultCRI cannot be changed if count of endpoints < 3
+  # podSubnetNodeCIDRPrefix cannot be changed in running cluster
```

Comment:
You can reduce the boilerplate and make the hook more performant by storing the result on the `jq` in the intermediate objects:

```sh
oldConfig=$(context::jq -r '.review.request.oldObject.data."cluster-configuration.yaml"' | base64 -d)
newConfig=$(context::jq -r '.review.request.oldObject.data."cluster-configuration.yaml"' | base64 -d)
```

and then use them like:

```sh
oldPrefix=$(yq eval '.podSubnetNodeCIDRPrefix' - <<< "$oldConfig")
newPrefix=$(yq eval '.podSubnetNodeCIDRPrefix' - <<< "$newConfig")

```